app/src/main/java/ch/epfl/skysync/screens/admin/AdminFlightDetailsScreen.kt

```diff
@@ -102,26 +110,60 @@ fun FlightDetailBottom(
     deleteClick: () -> Unit,
 ) {
   BottomAppBar(containerColor = lightGray) {
-    Row(
-        modifier = Modifier.fillMaxWidth(),
-        horizontalArrangement = Arrangement.SpaceBetween,
-        verticalAlignment = Alignment.CenterVertically) {
-          ClickButton(
-              text = "Delete",
-              onClick = { deleteClick() },
-              modifier = Modifier.fillMaxWidth(0.3f).testTag("DeleteButton"),
-              color = lightRed)
-          ClickButton(
-              text = "Edit",
-              onClick = { editClick() },
-              modifier = Modifier.fillMaxWidth(3 / 7f).testTag("EditButton"),
-              color = Color.Yellow)
-          ClickButton(
-              text = "Confirm",
-              onClick = { confirmClick() },
-              modifier = Modifier.fillMaxWidth(0.7f).testTag("ConfirmButton"),
-              color = lightGreen)
-        }
+    Row(modifier = Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
+      Spacer(modifier = Modifier.fillMaxWidth(0.05f))
+      Row(modifier = Modifier.fillMaxWidth(0.9f / 3 / (1 - 0.05f)).testTag("DeleteButton")) {
+        TextButton(
```

Comment:
You don't need to make such complicated calculation for the fillMaxWidth. You can use the weights. For example to have each rom taking the same space you could have `Modifier.fillMaxWidth.weight(1f)` for each row that you want of the same size. Note that, for it to work you will also need to embed this rows in a Row or a Column (which is the case here).